apps/web/pages/team/[slug].tsx

```diff
@@ -316,7 +319,9 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
       name: member.name,
       id: member.id,
       bio: member.bio,
+      subteams: member.subteams,
       username: member.username,
+      accepted: member.accepted,
```

Comment:
We now rely on accepted property to show members in the profile, as the username will be set even for invited people